luigi/templates/history.html

```diff
@@ -0,0 +1,117 @@
+<!--
+@Copyright 2015 Naver Corp.
+@Author Yeseul Park (yeseul.park@navercorp.com)
+-->
+
+<!-- Extend our site layout -->
+{% extends "layout.html" %}
+{% block content %}
+    <h1> {{name}} History </h1>
+	<br>
+	<div class="row">
+	<style>
+		#chart svg {
+			height: 800px;
+			margin-top: 80px;
+			font-family: Arial, sans-serif;
+			font-size: 9px;
+			overflow: scroll;
+		}
+		
+	</style>
+	<div id="chart">
+		<svg></svg>
+	</div>
+	{% if chartResults and tableResults %}
+	<!--h5> {{ chartResults }} and {{ tableResults }}</h5 -->
+	<script>
+		var data_results  =  unescape("{{ chartResults }}");
+		var data_str = String(data_results).replace(/&quot;/ig,'"');
+		var data = JSON.parse(data_str);
+		var running_data = data.RUNNING;
+		var done_data = data.DONE;
+		var failed_data = data.FAILED;
+		var all_results = unescape("{{ tableResults }}");
```

Comment:
Yes. And sorry, no test ==> no merge I think. :)

Looking forward to an iteration on this PR :)